tests/UI/campaigns/functional/BO/08_design/01_themeAndLogo/03_advancedCustomization/01_advancedCustomization.js

```diff
@@ -0,0 +1,277 @@
+      await expect(pageTitle).to.contains(themeAndLogoPage.pageTitle);
+    });
+
+    it('should go to \'Advanced Customization\' page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToAdvancedCustomizationPage_1', baseContext);
+
+      await themeAndLogoPage.goToSubTabAdvancedCustomization(page);
+
+      const pageTitle = await advancedCustomizationPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(advancedCustomizationPage.pageTitle);
+    });
+
+    it('should download theme', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'downloadTheme', baseContext);
+
+      filePath = await advancedCustomizationPage.downloadTheme(page);
+      const exist = await files.doesFileExist(filePath);
+      await expect(exist, 'Theme was not downloaded').to.be.true;
+    });
+
+    it('should click on upload child theme button', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'clickOnUploadChildTheme', baseContext);
+
+      const modalTitle = await advancedCustomizationPage.clickOnUploadChildThemeButton(page);
+      await expect(modalTitle, 'Modal \'Upload child theme\' is not displayed')
+        .to.contains('Drop your child theme archive here or select file');
+    });
+
+    it('should upload child theme', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'uploadChildTheme', baseContext);
+
+      await files.renameFile(filePath, renamedFilePath);
+      const uploadTheme = await advancedCustomizationPage.uploadTheme(page, renamedFilePath);
+      await expect(uploadTheme, 'Child theme is not uploaded')
+        .to.contains('The child theme has been added successfully.');
+    });
+
+    it('should close the modal', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'closeModal', baseContext);
+
+      const isModalClosed = await advancedCustomizationPage.closeModal(page);
+      await expect(isModalClosed, 'Modal not closed').to.be.true;
+    });
+
+    // remove the child_classic.zip from themes folder
+    it('should remove the zip file of the child theme from the themes folder', async function () {
+      await testContext
+        .addContextItem(this, 'testIdentifier', 'removeZipFileChildThemeFromThemesFolder', baseContext);
+
+      const generatedFilePath = await files.getFilePathAutomaticallyGenerated(themesPath, renamedFilePath);
+      await files.deleteFile(generatedFilePath);
+    });
+
+    it('should remove the zip file of the child theme from the temporary path', async function () {
+      await testContext
+        .addContextItem(this, 'testIdentifier', 'removeChildThemeFromTemporaryPath', baseContext);
+
+      await files.deleteFile(renamedFilePath);
+    });
+  });
+
+  // 2 - Use the child theme
+  describe('BO - Design - Theme & Logo - Advanced Customization : Use the child theme', async () => {
```

Comment:
No need to write the test path in all the describes
```suggestion
  describe('BO - Design - Theme & Logo - Advanced Customization : Use the child theme', async () => {
    describe('Use the child theme', async () => {
```